tests/test_msg_eos_signtx.py

```diff
@@ -648,7 +648,7 @@ def test_unknown_advanced(self):
                     num_actions=1),
                 self.action_unknown('acbdefghijkl', 'mnopqrstuvwx', binascii.unhexlify('AB' * i)))
 
-            print(i, binascii.hexlify(res.hash))
+            #print(i, binascii.hexlify(res.hash))
```

Comment:
why leave comments in code :(. I've seen this everywhere.

Reply:
good call